### ISSUE TITLE
Tests seem to not work as intended.

### ISSUE THREAD

**Issue by yehudabortz** [learn-co-curriculum/sinatra-fwitter-group-project#75]
It appears that the test structure is very delicate and does not allow for the app to be written outside of the exact code written in the solutions branch, despite the app functioning as intended. 
Best, 
Yehuda

**Comment by drewkeat** [comment]
Agreed.  Tests seem to require explicitly specific logic structures to pass.  Specifically, the "Tweet Edit" tests.
